conbench/entities/run.py

```diff
@@ -278,6 +278,13 @@ def commit_hardware_run_map():
 
 
 class GitHubCreate(marshmallow.Schema):
+    @marshmallow.pre_load
+    def change_pr_number_empty_string_to_none(self, data, **kwargs):
+        if "pr_number" in data:
+            data["pr_number"] = data["pr_number"] or None
```

Comment:
Quick note: that is not as precise as the method name suggests :)

I'd do:
```
if data.get("pr_number") == "":
    data["pr_number] = None
```
because `data["pr_number"]` may evaluate to False for more values than just `""`.